### PR TITLE
ui: Don't show UI toggle on signin page (PROJQUAY-4402)

### PR DESCRIPTION
**Pull request by HammerMeetNail**
* Hide toggle when on signin page

### CODE REVIEW

File: static/directives/header-bar.html [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
         <!-- Normal -->
         <ul class="nav navbar-nav navbar-right hidden-xs" ng-switch on="user.anonymous">
-          <li class="new-ui-toggle" ng-if="Features.UI_V2">
+          <li class="new-ui-toggle" ng-if="Features.UI_V2 && checkElementByID('signin-username') !== true">
```
Comment (syed) [inline static/directives/header-bar.html:47]: you don't need the `!== true` it's already a boolean just the return value will suffice

File: static/js/directives/ui/header-bar.js [hunk 2]
```diff
@@ -145,6 +145,14 @@ angular.module('quay').directive('headerBar', function () {
           'namespace': namespace
         };
       };
+
+      $scope.checkElementByID = function(elementID) {
+        var results = false;
```
Comment (syed) [inline static/js/directives/ui/header-bar.js:150]: `results` suggest a multiple value like an array. `result` or even `exists` should work

File: static/directives/header-bar.html (hunk 1 above)
Comment (HammerMeetNail) [reply static/directives/header-bar.html]: Fixed

File: static/js/directives/ui/header-bar.js (hunk 2 above)
Comment (HammerMeetNail) [reply static/js/directives/ui/header-bar.js]: Fixed

**Review by syed — APPROVED** [review]
LGTM

**Pull request merged**